web/portal/src/main/webapp/WEB-INF/conf/portal/portal/sharedlayout.xml

```diff
@@ -61,17 +61,23 @@
       <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
       <factory-id>addonContainer</factory-id>
     </container>
-    <container id="middle-topNavigation-container" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
-      <name>middle-topNavigation-container</name>
+    <container id="ChatPortlet" template="system:/groovy/portal/webui/container/UIAddOnContainer.gtmpl">
```

Comment:
Chat is not includeed in Meeds. it's an eXo Platform feature.  We should extend this in eXo, not in Meeds.